src/Slices/ServiceInventory/UI/Components/RowActionsMenu/DestroyAction/DestroyAction.tsx

```diff
@@ -32,13 +30,32 @@ export const DestroyAction: React.FC<Props> = ({
     version,
   });
   const onSubmit = async () => {
-    setIsOpen(false);
+    closeModal();
     const result = await trigger(refetch);
 
     if (Maybe.isSome(result)) {
       setErrorMessage(result.value);
     }
   };
+  const openModal = () => {
```

Comment:
be consistent with the docstrings

Reply:
won't Service Inventory won't be replaced by your redesign?

Reply:
Doesn't mean the current code shouldn't be clean and aligned. The redesign will remove the collapsible content, and replace the buttons, yes, but that is for a later step.